O365WebHook/formatO365Log.js

```diff
@@ -0,0 +1,33 @@
+const Parse = require('al-collector-js').Parse;
+
+module.exports = function(item) {
+    //Paths from https://docs.microsoft.com/en-us/office/office-365-management-api/office-365-management-activity-api-schema#common-schema
+    const creationTimePaths = [
+        {path: ['CreationTime']}
+    ];
+    const messageTypeIdPaths = [
+        {path: ['Operation']}
+    ];
+
+    let message;
+    try {
+        message = JSON.stringify(item);
+    }
+    catch(err) {
+        return callback(`Unable to stringify content. ${err}`);
+    }
+
+    let creationTime = Parse.getMsgTs(item, creationTimePaths);
+    let messageTypeId = Parse.getMsgTypeId(item, messageTypeIdPaths);
+
+    return {
+        messageTs: creationTime.sec,
+        priority: 11,
+        progName: 'o365webhook',
+        pid: undefined,
+        message: message,
+        messageType: 'json/azure.o365',
+        messageTypeId: messageTypeId,
+        messageTsUs: undefined
```

Comment:
Could you please check Azure docs. Most likely we can add microseconds here, `creationTime.usec`